src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1733,6 +1734,103 @@ <h2 class="text--center">Balance sheet</h2>
               </tr>
               {{end}}
 
+              <!-- BALANCE SHEET - CURRENT ASSETS -->
+
+              {{if $currentAssets := $balanceSheet.current_assets}}
+
+                <tr>
+                  <th class = "strong" id="current-assets-heading">Current assets</th>
+                </tr>
+
+                <!-- BALANCE SHEET - CURRENT ASSETS - STOCKS -->
+
+              {{if $stocks := $currentAssets.stocks}}
+                <tr class="figures">
+                  <td id="stocks-label">Stocks:</td>
+
+                  <td class="figure noteIndex" id="uk-busTotalInventoriesCY_ENDNoteIndexes"></td>
```

Comment:
Do we know when we are adding the ordering number? they are in the template; however, this has not been added to any of the notes yet.

Reply:
These will be when we start adding the corresponding balance sheet notes